### DISCUSSION TITLE
support multiple key_prefix in s3

### DISCUSSION THREAD

**Discussion by tooptoop4** [drakkan/sftpgo#1091] [General]
want user to see folderA/ and folderC/ but not folderB/

**Comment by drakkan** [comment]
Hello,

you have opened 12 issues and 6 discussions in this repository. I have also implemented some features you requested. I have indicated to you the support policy several times. Please understand I have limited time and need to do paid jobs to pay my bills, I don't have the time/motivation to support users who just ask without giving anything back, sorry. If all users behave like this the project will die or I will be forced to turn it into a proprietary application. Thanks for understanding
